tests/.gitignore

```diff
@@ -1,2 +0,0 @@
-compiled_scripts/
-template_reference_data/
```

Comment:
Why was this file deleted?

Reply:
That was a severe mistake from my side, sorry for that.